### PR TITLE
Update dumpling-overview.md

### PR DESCRIPTION
**Pull request by pepezzzz**
add full backup ftwrl details

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add full backup ftwrl details
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [] v3.1 (TiDB 3.1 versions)
- [] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dumpling-overview.md [hunk 1]
```diff
@@ -258,7 +258,7 @@ Dumpling 也可以通过 `-B` 或 `-T` 选项导出特定的数据库/数据表
 
 Dumpling 通过 `--consistency <consistency level>` 标志控制导出数据“一致性保证”的方式。对于 TiDB 来说，默认情况下，会通过获取某个时间戳的快照来保证一致性（即 `--consistency snapshot`）。在使用 snapshot 来保证一致性的时候，可以使用 `--snapshot` 选项指定要备份的时间戳。还可以使用以下的一致性级别：
 
-- `flush`：使用 [`FLUSH TABLES WITH READ LOCK`](https://dev.mysql.com/doc/refman/8.0/en/flush.html#flush-tables-with-read-lock) 来保证一致性。
+- `flush`：使用 [`FLUSH TABLES WITH READ LOCK`](https://dev.mysql.com/doc/refman/8.0/en/flush.html#flush-tables-with-read-lock) 短暂地中断备份库的 DML 和 DDL 操作，保证并发备份连接的全局一致性和记录 pos 信息，所有的并发备份连接建立后释放该锁。推荐在业务低峰或者 MySQL 备库上进行全量备份。
```
Comment (lichunzhu) [inline dumpling-overview.md:261]: 所有的并发备份连接建立后释放该锁。
该句不准确，释放锁的时机目前由 `--transactional-consistency` 控制。

Comment (pepezzzz) [reply dumpling-overview.md]: 所有的并发备份连接建立后释放该锁->所有的并发备份连接启动事务后释放该锁  ?

File: dumpling-overview.md [hunk 2]
```diff
@@ -258,7 +258,7 @@ Dumpling 也可以通过 `-B` 或 `-T` 选项导出特定的数据库/数据表
 
 Dumpling 通过 `--consistency <consistency level>` 标志控制导出数据“一致性保证”的方式。对于 TiDB 来说，默认情况下，会通过获取某个时间戳的快照来保证一致性（即 `--consistency snapshot`）。在使用 snapshot 来保证一致性的时候，可以使用 `--snapshot` 选项指定要备份的时间戳。还可以使用以下的一致性级别：
 
-- `flush`：使用 [`FLUSH TABLES WITH READ LOCK`](https://dev.mysql.com/doc/refman/8.0/en/flush.html#flush-tables-with-read-lock) 来保证一致性。
+- `flush`：使用 [`FLUSH TABLES WITH READ LOCK`](https://dev.mysql.com/doc/refman/8.0/en/flush.html#flush-tables-with-read-lock) 短暂地中断备份库的 DML 和 DDL 操作，保证并发备份连接的全局一致性和记录 pos 信息，所有的并发备份连接启动事务后释放该锁。推荐在业务低峰或者 MySQL 备库上进行全量备份。
```
Comment (TomShawn) [inline dumpling-overview.md:261]: "并发备份连接"整体是一个名词吗？
```suggestion
- `flush`：使用 [`FLUSH TABLES WITH READ LOCK`](https://dev.mysql.com/doc/refman/8.0/en/flush.html#flush-tables-with-read-lock) 短暂地中断备份库的 DML 和 DDL 操作、保证并发备份连接的全局一致性和记录 POS 信息。所有的并发备份连接启动事务后释放该锁。推荐在业务低峰或者 MySQL 备份库上进行全量备份。
```

Comment (lichunzhu) [reply dumpling-overview.md:261]: 不是，并发的备份连接，或者不要并发也行，大概就是数个备份连接

Comment (TomShawn) [reply dumpling-overview.md:261]: Updated

**Pull request merged**